components/scheduler/views/day.md

```diff
@@ -10,7 +10,7 @@ position: 1
 
 # Day View
 
-The Day view of the Scheduler for Blazor shows a single day to the user.
+The Agenda view of the Scheduler for Blazor shows a weekly summary (or a custom period set by the user) in a table format. Also the Day view can be configured to display the events that are [grouped by resource]({%slug scheduler-resource-grouping%}).
```

Comment:
Right now, this paragraph is somewhat confusing.

Is this the Agenda or the Day view? And why would you be talking about the Agenda view in a section titled Day View? Also, why would you talk about a weekly summary in a section titled Day View?

 Again, go with active voice for the second sentence and explain who can do the configuring.

Reply:
It seems like the agenda view information here was mistakenly added. Good catch and thanks.